v2.1/range-merges.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Range Merges
+summary: To help improve performance, CockroachDB can automatically merge small ranges of data together.
+toc: true
+---
+
+To help improve your cluster's performance, CockroachDB can automatically merge small ranges of data together to form fewer, larger ranges.
+
+{{site.data.alerts.callout_danger}}
+
+In CockroachDB 2.1, range merges are an [experimental feature](experimental-features.html).
+
+Only use this feature if your entire cluster is running CockroachDB 2.1.3 or newer.
+
+{{site.data.alerts.end}}
+
+## Settings
+
+### Enable/disable range merges
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_merge.queue_enabled` to:
+
+- `true` to enable range merges
+- `false` to disable range merges
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.range_merge.queue_enabled = true;
+~~~
+
+#### When to disable range merges
+
+Disable (or do not use) range merges if you:
+
+- Want to manually control range splits using [`SPLIT AT`](split-at.html)
+
+- Use CockroachDB 2.1.2 or older
+
+#### When to enable range merges
+
+Range merges can provide performance improvements if you:
+
+- Use CockroachDB 2.1.3 or newer
+
+## How range merges work
+
+### Overview
+
+CockroachDB splits your cluster's data into many ranges (64MiB by default), which are defined by a range of keys they contain. For example, your cluster might contain a range that contains only customers whose ID is between `[1000, 2000)`. Once that range of keys constitutes more than 64MiB of data, the range is split into two 32MiB ranges.
+
+However, as you delete data from your cluster, a range might contain far less data. Over the lifetime of a cluster, this could lead to a large number of small ranges.
+
+To reduce the number of small ranges, your cluster can have any range below a certain threshold (16MiB by default) try to merge with its "neighbor", i.e. the range with the next-highest key. Using our example above, this might be the range whose keys are `[2000, 3000)`.
```

Comment:
I find this usage of "next-highest" confusing. Given 1 2 3, if you have 2, the "next highest" is 1, not 3. Right? 

Here's a straw man alternative: "...try to merge with its "neighbor," i.e., the range that starts where this range ends."

If we use this "next-highest" terminology elsewhere I'll rescind my complaint, though, in the interest of consistency.